### ISSUE TITLE
[Bug]: APACHE2: parse error: Expected string key before ':' at line 1, column 4

### ISSUE THREAD

**Issue by Rogelage** [FaserF/hassio-addons#94]
### The problem

Since the last update V2.4.5, the apache2 server has stopped working. I've gone back to V2.4.4 but it doesn't work either.
Error:
parse error: Expected string key before ':' at line 1, column 4
[07:02:33] ERROR: Unknown HTTP error occured
You haven't put your website to null
A default website will now be used
find: null: No such file or directory

### What version of Home Assistant Core has the issue?

2022.9.7

### What was the last working version of Home Assistant Core?

2022.9.7

### What type of installation are you running?

Home Assistant Supervised

### Add-On causing the issue

Apache2

### Log information

```txt
parse error: Expected string key before ':' at line 1, column 4
[07:02:33] ERROR: Unknown HTTP error occured
You haven't put your website to null
A default website will now be used
find: null: No such file or directory
```

**Comment by FaserF** [comment]
Unfortunatly I am unable to reproduce it. Could you please remove your web files from your /share folder and then upload it there again and reinstall the addon? Please also double check that you have set your web folder path in the addon configuration variables. Null means that the variable for the folder path was empty.

**Comment by Rogelage** [comment]
I have reinstalled the addon, I have changed /share folder. I have deleted the website and put it back. But it still doesn't work.
I have also restarted HA, I don't know what else to do...

**Comment by stevnhoved** [comment]
I have the the exactly same problem with update (or clean install) of Apache2 Minimal v2.4.5. It also didn’t helped to just restore the backup of v.2.4.2.

I finally solved it by completely uninstalling the add-on first, and then restoring the backup of v2.4.2.

**Comment by Rogelage** [comment]
Thank you very much for your reply. If I don't have a backup, how can I install a previous version?

**Comment by FaserF** [comment]
Could you tell me on which Hardware you are using the addon?

**Comment by Rogelage** [comment]
Raspberry pi 4

**Comment by FaserF** [comment]
Could you please try to install for example my wiki.js addon. I want to know if my last changes to all addons about the docker installation is the problem or if there is only a problem with the apache2 addon

**Comment by stevnhoved** [comment]
RPI 4 too.

Wiki.JS only complains abort missing MariaDB.

**Comment by Rogelage** [comment]
Wiki.JS works perfectly.

s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
This add-on requires the MariaDB core add-on 2.0 or newer!
Starting Wiki.JS
Loading configuration from /wiki/config.yml... OK

**Comment by djtail** [comment]
having the same problem, after the update

parse error: Expected string key before ':' at line 1, column 4
[19:19:01] ERROR: Unknown HTTP error occured

**Comment by FaserF** [comment]
fixed with release 2.4.6
